platforms/android/library/src/main/java/io/element/android/wysiwyg/utils/HtmlToSpansParser.kt

```diff
@@ -314,11 +315,11 @@ internal class HtmlToSpansParser(
             ?: LinkDisplay.Plain
         when(linkDisplay) {
             is LinkDisplay.Custom -> {
-                val span = linkDisplay.customSpan
+                val span = CustomReplacementSpan(linkDisplay.customSpan)
                 replacePlaceholderWithPendingSpan(last.span, span, last.start, text.length, Spanned.SPAN_EXCLUSIVE_EXCLUSIVE)
             }
             LinkDisplay.Pill -> {
-                val span = PillSpan((last.span).link, resourcesHelper.getColor(styleConfig.pill.backgroundColor))
+                val span = PillSpan(resourcesHelper.getColor(styleConfig.pill.backgroundColor))
```

Comment:
- I don't know kotlin, so perhaps I'm missing something, but I don't think I can see the signature to `PillSpan` changing in this PR. Was this incorrect before and is being updated to the correct signature now? Or am I missing something obvious? Or, option C, am I'm completely misunderstanding the change?

Reply:
I spotted that the `url` field of `PillSpan` was not used so have removed in this PR ([link to that change](https://github.com/matrix-org/matrix-rich-text-editor/pull/677/files#diff-fe66889a904ee3e8f6d18fb80b16848051764003c6f93f88b7471f2edd6fc843L11))